precompiles/wasmd/wasmd.go

```diff
@@ -147,6 +147,10 @@ func (p Precompile) RunAndCalculateGas(evm *vm.EVM, caller common.Address, calli
 	}
 	ctx = ctx.WithGasMeter(sdk.NewGasMeterWithMultiplier(ctx, gasLimitBigInt.Uint64()))
 
+	if logger != nil && logger.OnGasChange != nil {
```

Comment:
you might want to rebase; we recently refactored `RunAndCalculateGas` logics and these new lines should probably go under https://github.com/sei-protocol/sei-chain/blob/main/precompiles/common/precompiles.go#L143

Reply:
Yeah I'll bring in latest changes and perform the necessary requested changes.